lib/octicons_gem/lib/octicons_v2/octicon_v2.rb

```diff
@@ -84,5 +81,25 @@ def calculate_width(height)
     def calculate_height(width)
       (width.to_i * @height) / @width
     end
+
+    def get_octicon(symbol, options = {})
+      if octicon = OcticonsV2::OCTICON_SYMBOLS[symbol]
+        height = options[:height] || options[:width] || 16
```

Comment:
What do you think about deprecating the `width` option?

Reply:
```suggestion
        height = options[:height] || options[:width] || DEFAULT_HEIGHT
```

Generally I think it's best to not have magic numbers. Might we assign a constant for `16`?

Reply:
Makes sense. Where should `DEFAULT_HEIGHT` be defined?

Reply:
In this class, I think.